src/test/mock/clitools.rs

```diff
@@ -668,8 +668,11 @@ impl Config {
             println!("expected.ok: true");
             print_indented("expected.stdout", stdout);
             print_indented("expected.stderr", stderr);
-            dbg!(out.stdout == stdout);
```

Comment:
IMO `dbg!()` in test-only code can actually be pretty useful, since it will typically only be shown when a test fails anyway. So I would recommend that we `allow(dbg_macro)` for all of the `test` module.

(Apparently clippy already allows its use directly in tests.)

Reply:
Addressed in #4063.